### PR TITLE
Add docs and images for PMSA003i

### PR DESCRIPTION
**Pull request by sjtrny**
## Description:

Adds docs for pmsa003i component https://github.com/esphome/esphome/pull/1501

## Checklist:

  - [x] Branch: `next` is for changes and new documentation that will go public with the next ESPHome release. Fixes, changes and adjustments for the current release should be created against `current`.
  - [x] Link added in `/index.rst` when creating new documents for new components or cookbook.

### CODE REVIEW

File: components/sensor/pmsa003i.rst [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  All options from :ref:`Sensor <config-sensor>`.
+- **pmc_1_0** (**Optional**): Count of particles with diameter > 1 um in 0.1 L of air (#/0.1L).
+  All options from :ref:`Sensor <config-sensor>`.
+- **pmc_2_5** (**Optional**): Count of particles with diameter > 2.5 um in 0.1 L of air (#/0.1L).
+  All options from :ref:`Sensor <config-sensor>`.
+- **pmc_5_0** (**Optional**): Count of particles with diameter > 5 um in 0.1 L of air (#/0.1L).
+  All options from :ref:`Sensor <config-sensor>`.
+- **pmc_10_0** (**Optional**): Count of particles with diameter > 10 um in 0.1 L of air (#/0.1L).
+  All options from :ref:`Sensor <config-sensor>`.
+- **standard_units** (*Optional*, bool): Whether to use standard units (`True`) or environmental units (`False`).
```
Comment (glmnet) [inline components/sensor/pmsa003i.rst]:
```suggestion
- **standard_units** (*Optional*, bool): ``True`` to use standard units or ``False`` to use  environmental units. Defaults to ``True``.
```

File: components/sensor/pmsa003i.rst [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+          name: "PMC <1µm"
+        pmc_2_5:
+          name: "PMC <2.5µm"
+        pmc_5_0:
+          name: "PMC <5µm"
+        pmc_10_0:
+          name: "PMC <10µm"
+        address: 0x12
+        update_interval: 60s
+        standard_units: True
```
Comment (glmnet) [inline components/sensor/pmsa003i.rst]:
```suggestion
```

Remove default values.

**Review by glmnet — APPROVED** [review]
Looks good

File: components/sensor/pmsa003i.rst [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+          name: "PMC <2.5µm"
+        pmc_5_0:
+          name: "PMC <5µm"
+        pmc_10_0:
+          name: "PMC <10µm"
+
+Configuration variables:
+------------------------
+
+- **pm_1_0** (**Optional**): Mass of particles with a diameter of 1 micrometres or less (μg/m^3).
```
Comment (OttoWinter) [inline components/sensor/pmsa003i.rst:48]:
```suggestion
- **pm_1_0** (*Optional*): Mass of particles with a diameter of 1 micrometres or less (μg/m^3).
```

File: components/sensor/pmsa003i.rst [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+- **pmc_0_5** (**Optional**): Count of particles with diameter > 0.5 um in 0.1 L of air (#/0.1L).
+  All options from :ref:`Sensor <config-sensor>`.
+- **pmc_1_0** (**Optional**): Count of particles with diameter > 1 um in 0.1 L of air (#/0.1L).
+  All options from :ref:`Sensor <config-sensor>`.
+- **pmc_2_5** (**Optional**): Count of particles with diameter > 2.5 um in 0.1 L of air (#/0.1L).
+  All options from :ref:`Sensor <config-sensor>`.
+- **pmc_5_0** (**Optional**): Count of particles with diameter > 5 um in 0.1 L of air (#/0.1L).
+  All options from :ref:`Sensor <config-sensor>`.
+- **pmc_10_0** (**Optional**): Count of particles with diameter > 10 um in 0.1 L of air (#/0.1L).
+  All options from :ref:`Sensor <config-sensor>`.
```
Comment (OttoWinter) [inline components/sensor/pmsa003i.rst]:
```suggestion
- **pm_1_0** (*Optional*): Mass of particles with a diameter of 1 micrometres or less (μg/m^3).
  All options from :ref:`Sensor <config-sensor>`.
- **pm_2_5** (*Optional*): Mass of particles with a diameter of 2.5 micrometres or less (μg/m^3).
  All options from :ref:`Sensor <config-sensor>`.
- **pm_10_0** (*Optional*): Mass of particles with a diameter of 10 micrometres or less (μg/m^3).
  All options from :ref:`Sensor <config-sensor>`.
- **pmc_0_3** (*Optional*): Count of particles with diameter > 0.3 um in 0.1 L of air (#/0.1L).
  All options from :ref:`Sensor <config-sensor>`.
- **pmc_0_5** (*Optional*): Count of particles with diameter > 0.5 um in 0.1 L of air (#/0.1L).
  All options from :ref:`Sensor <config-sensor>`.
- **pmc_1_0** (*Optional*): Count of particles with diameter > 1 um in 0.1 L of air (#/0.1L).
  All options from :ref:`Sensor <config-sensor>`.
- **pmc_2_5** (*Optional*): Count of particles with diameter > 2.5 um in 0.1 L of air (#/0.1L).
  All options from :ref:`Sensor <config-sensor>`.
- **pmc_5_0** (*Optional*): Count of particles with diameter > 5 um in 0.1 L of air (#/0.1L).
  All options from :ref:`Sensor <config-sensor>`.
- **pmc_10_0** (*Optional*): Count of particles with diameter > 10 um in 0.1 L of air (#/0.1L).
  All options from :ref:`Sensor <config-sensor>`.
```

Only required items are bold

**Review by OttoWinter — APPROVED** [review]
One small consistency thing, looks great otherwise!

**Pull request merged**